html/elements/script.json

```
@@ -558,13 +558,16 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "61"
+                  "version_added": "61",
+                  "notes": "Chrome does not load module scripts without the <code>async</code> attribute when the page is served as XHTML (<code>application/xhtml+xml</code>) - <a href='https://crbug.com/717643'>Chromium Issue #717643</a>"
```

We usually format bug links more minimally, like this:

```suggestion
                  "notes": "Chrome does not load module scripts without the <code>async</code> attribute when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
```

---

My apologies, I had actually more or less copied the format that some of the implementation notes for "defer" had, should those be restyled too?

---

Sorry, I didn't realize you had copied from elsewhere. There's no need to change the old notes in this PR.